pkg/virt-controller/services/template.go

```diff
@@ -612,6 +626,14 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		nodeSelector[k] = v
 
 	}
+	if IsCPUNodeDiscoveryEnabled(t.configMapStore) {
+		if vmi.Spec.Domain.CPU != nil && vmi.Spec.Domain.CPU.Model != "" {
+			if vmi.Spec.Domain.CPU.Model != v1.CPUModeHostModel && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostPassthrough {
+				nodeSelector[NFD_CPU_FAMILY_PREFIX+vmi.Spec.Domain.CPU.Model] = "true"
```

Comment:
`NFD_CPU_FAMILY_PREFIX+vmi.Spec.Domain.CPU.Model` this should be a function somewhere. We want this important part ot be isolated somehwere. i.e. `CPUFamilyLabelFromCPUFamily`